backend/migrations/20221212000000-search-indexed.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+    dbm = options.dbmigrate;
+    type = dbm.dataType;
+    seed = seedLink;
+};
+
+exports.up = async function(db) {
+    await db.runSql(`
+    alter table content_source
+      add column indexed tinyint(1) not null default 0,
+      add index idx_indexed (indexed);  
```

Comment:
The better index would be `alter table content_source add index idx_indexed_content_source_id (indexed, content_source_id);` (since both columns are used in the query).

Reply:
Fixed.